src/types/JSONSchema.ts

```diff
@@ -126,6 +127,10 @@ export const getRootSchema = memoize((schema: LinkedJSONSchema): LinkedJSONSchem
   return getRootSchema(parent)
 })
 
+export function isBoolean(schema: LinkedJSONSchema | JSONSchemaType): schema is boolean {
```

Comment:
Maybe this is sort of an overengineered approach, but I couldn't think in anything better